### ISSUE TITLE
Windows app to record time in programmes and quickly add them to Kimai or add support of memtime to import function

### ISSUE THREAD

**Issue by EduNetEurope** [kimai/kimai#4364]
### Describe the problem

We are so happy we found Kimai. It is a great app.

The only thing we miss is a Windows app that records our time in Windows programmes and allows us to select projects / activities for the entries to automatically add them to Kimai. Clockify has such an app. Memtime unfortunately is not connected to Kimai at the moment.

### Describe the solution you'd like

Windows app that records time in Windows programmes and allows us to add time entries via API.

### Describe alternatives you've considered

Alternative inspired by https://github.com/kimai/kimai/issues/4126:
Add ability to add memtime CSV to import function.

**Comment by kevinpapst** [comment]
Please attach a CSV from memtime, so I can test it

**Comment by EduNetEurope** [comment]
Hi @kevinpapst,

Memtime supports timesheet export in both CSV and XLSX formats. Exporting in CSV format causes issues with special characters. Would it be possible to enable imports from XLSX files? Unfortunately, we use — (ALT+0151) and other characters in both project and activity names.

Please find the exports
[memtime export csv.csv](https://github.com/user-attachments/files/17804385/memtime.export.csv.csv)
[memtime export.xlsx](https://github.com/user-attachments/files/17804386/memtime.export.xlsx)
 attached for your reference.
 
 Thanks for looking into it.

**Comment by kevinpapst** [comment]
> Exporting in CSV format causes issues with special characters

What does that mean? Export in memtime doesn't work and therefor Kimi should add a importer for XLSX?

> Would it be possible to enable imports from XLSX files

I have never heard about such a flag. Excel is a proprietary format and was NEVER made for machine data transfer.

**Comment by EduNetEurope** [comment]
It was just a question. CSV does not like the special characters, I don't think it is a memtime-related issue. We will need to rename the projects to avoid the issue.

**Comment by kevinpapst** [comment]
Have you asked memtime first? Why should any special character cause problems in CSV? Never heard of such a limitation.

**Comment by EduNetEurope** [comment]
I don't think it is a memtime issue. When I download the Excel file and save it as a CSV, the "—" we use will also be turned into ???.
The long dash also appears as ??? in Clockify CSV exports. It seems to be a CSV restriction. We will change our labelling system.

**Comment by kevinpapst** [comment]
Save it as UTF-8.

**Comment by EduNetEurope** [comment]
Tried that as well. Will try again. Thanks for the tipp.
